src/lib/y2network/interfaces_collection.rb

```diff
@@ -87,5 +96,70 @@ def ==(other)
     end
 
     alias_method :eql?, :==
+
+    # Returns the interfaces that are enslaved in the given bridge
+    #
+    # @param master [String] bridge name
+    # @return [Array<String>] a list of interface names
```

Comment:
`Array<String>` ? I would have expected an `Array<Interface>` (or an `InterfacesCollection`).

Reply:
I'd propose to add it as another TODO bcs I planed to rework this method when moving to proper class as written in the todo.